packages/lab/src/tree/Tree.css

```diff
@@ -1,26 +1,24 @@
 .uitkTree {
-  --uitkTreeNode-height: var(--uitkTree-node-height, var(--uitk-size-stackable-unit));
-
-  --tree-node-collapse: var(--uitkTree-toggle-collapse, var(--svg-tree-node-collapse));
-  --tree-node-expand: var(--uitkTree-toggle-expand, var(--svg-tree-node-expand));
-  --tree-toggle-width: 12px;
+  --tree-borderColor-focusVisible: rgb(141, 154, 179);
+  --tree-node-collapse: var(--uitkTree-node-collapse, var(--list-svg-tree-node-collapse));
+  --tree-node-expand: var(--uitkTree-node-expand, var(--list-svg-tree-node-expand));
   --tree-node-expanded-transform: var(--uitkTree-node-expanded-transform, none);
   --tree-node-indent: 0px;
+  --tree-toggle-width: 12px;
 
-  --list-hilited-bg: var(--hw-list-hilited-bg, rgba(0, 0, 0, 0.1));
-  --list-item-height: var(--hw-list-item-height, 30px);
-  --list-item-padding: var(--hw-list-item-padding, 0 6px);
-  --list-item-header-bg: var(--hw-list-item-header-bg, black);
-  --list-item-header-color: var(--hw-list-item-header-color, white);
+  /* TODO: Use appropriate characteristics */
```

Comment:
Where do we document these requirements, an existing jira, or a new github issue? I imagine this is a todo for the designers?

Reply:
this is just for us to implement the styling correctly according to the characteristics file.. the comment isn't really necessary since it applied to all lab components anyway